.github/workflows/pre.yml

```diff
@@ -69,7 +69,7 @@ jobs:
           flags: unit
           name: unit
           verbose: true
-          fail_ci_if_error: true
+          fail_ci_if_error: false
```

Comment:
add a TODO like change this back when we find a way to stabilize it?

Reply:
That's a good point to discuss on, I prefer leave it be to false.  And if codecov continously being unstable we transfer to another tool.